t/coretests.pm

```diff
@@ -338,7 +343,8 @@ SKIP: 	{
 	ok($version == $new_version, '$version == $new_version');
 	skip "version require'd instead of use'd, cannot test declare", 1
 	    unless defined $qv_declare;
-	$version = &$qv_declare(1.2.3);
+
+    $version = main->can($qv_declare)->(1.2.3);
```

Comment:
I think this is indented incorrectly

Reply:
fixed via 590c6182... unfortunately the whole indentation of this file is a mess... mixing spaces and tabs...
restored the "\t\s+" indentation used around the other lines to match